packages/loot-design/patches/downshift+1.31.14.patch

```diff
@@ -1,21 +0,0 @@
-diff --git a/node_modules/downshift/dist/downshift.esm.js b/node_modules/downshift/dist/downshift.esm.js
```

Comment:
This patch was simply deleted?

Reply:
I need to double check this one - Yarn workspaces is really conflicting with the patching - I think it was trying to patch something before it was installed - or there might be some symlink weirdness that's occurring on Linux.

If you disable CI detection, everything appears to work. But I'm not a fan of that. I wonder if Circleci was using an older version of CI detection, so the patcher didn't actually detect CI.